### PR TITLE
feature/mapping-profiles

### PR DESCRIPTION
**Pull request by kalachck**

### CODE REVIEW

File: StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/CommentDto.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿namespace PBJ.StoreManagementService.Business.Dtos
+{
+    public class CommentDto
+    {
+        public string? Content { get; set; }
+
+        public DateTime CommentDate { get; set; }
```
Comment (nantonov) [inline StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/CommentDto.cs]: It difficult to understand what does it mean
Date when comment was created? => CreatedAt
Published? => PublishedAt
and so on...

File: StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/CommentDto.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿namespace PBJ.StoreManagementService.Business.Dtos
+{
+    public class CommentDto
```
Comment (nantonov) [inline StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/CommentDto.cs:3]: Why this DTO doesn't have an ID?

File: StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/FollowingDto.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿namespace PBJ.StoreManagementService.Business.Dtos
+{
+    public class FollowingDto
```
Comment (nantonov) [inline StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/FollowingDto.cs]: Strange DTO, why not in the User?

File: StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/FollowingDto.cs [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+    {
+        public DateTime FollowingDate { get; set; }
```
Comment (nantonov) [inline StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/FollowingDto.cs]: The same, use 'At' postfix with dates

File: StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/FollowingDto.cs [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+        public int FollowingUserId { get; set; }
+
+        public UserDto? FollowingUser { get; set; }
+
+        public ICollection<UserFollowingDto>? UserFollowings { get; set; }
```
Comment (nantonov) [inline StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/FollowingDto.cs]: Can we use IReadOnlyCollection here?

File: StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/PostDto.cs [hunk 6]
```diff
@@ -0,0 +1,15 @@
+﻿namespace PBJ.StoreManagementService.Business.Dtos
+{
+    public class PostDto
```
Comment (nantonov) [inline StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/PostDto.cs:3]: Why without ID?

File: StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/SubscriptionDto.cs [hunk 7]
```diff
@@ -0,0 +1,13 @@
+﻿namespace PBJ.StoreManagementService.Business.Dtos
+{
+    public class SubscriptionDto
```
Comment (nantonov) [inline StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/SubscriptionDto.cs]: Strange, why not in the user?

File: StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/SubscriptionDto.cs [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+    {
+        public DateTime SubscriptionDate { get; set; }
```
Comment (nantonov) [inline StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/SubscriptionDto.cs]: Postfix 'At'

File: StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/PostDto.cs [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+    {
+        public string? Content { get; set; }
+
+        public DateTime PostDate { get; set; }
```
Comment (nantonov) [inline StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/PostDto.cs]: Postfix 'At'

File: StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/PostDto.cs [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+
+        public int UserId { get; set; }
+
+        public UserDto? User { get; set; }
+
+        public ICollection<CommentDto>? Comments { get; set; }
```
Comment (nantonov) [inline StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/PostDto.cs]: ReadonlyCollection?

File: StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/SubscriptionDto.cs [hunk 11]
```diff
@@ -0,0 +1,13 @@
+﻿namespace PBJ.StoreManagementService.Business.Dtos
+{
+    public class SubscriptionDto
+    {
```
Comment (nantonov) [inline StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/SubscriptionDto.cs]: Where is ID?

File: StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/UserDto.cs [hunk 12]
```diff
@@ -0,0 +1,29 @@
+﻿using PBJ.StoreManagementService.DataAccess.Entities;
+
+namespace PBJ.StoreManagementService.Business.Dtos
+{
+    public class UserDto
```
Comment (nantonov) [inline StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/UserDto.cs:3]: Where is ID?

File: StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/UserDto.cs [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+    {
+        public string? Name { get; set; }
+
+        public string? Surname { get; set; }
+
+        public string? Lastname { get; set; }
+
+        public DateTime Birthdate { get; set; }
+
+        public string? Login { get; set; }
```
Comment (nantonov) [inline StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/UserDto.cs]: What does it mean?

File: StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/UserFollowingDto.cs [hunk 14]
```diff
@@ -0,0 +1,13 @@
+﻿namespace PBJ.StoreManagementService.Business.Dtos
+{
+    public class UserFollowingDto
```
Comment (nantonov) [inline StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/UserFollowingDto.cs]: Id?

File: StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/UserSubscriptionDto.cs [hunk 15]
```diff
@@ -0,0 +1,13 @@
+﻿namespace PBJ.StoreManagementService.Business.Dtos
+{
+    public class UserSubscriptionDto
```
Comment (nantonov) [inline StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/UserSubscriptionDto.cs]: Id?

**Review by nantonov — CHANGES_REQUESTED** [review]

File: StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/UserDto.cs (hunk 13 above)
Comment (kalachck) [reply StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/UserDto.cs]: Like email or personal unique nickname

File: StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/SubscriptionDto.cs (hunk 7 above)
Comment (kalachck) [reply StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/SubscriptionDto.cs]: don't get it, what do you mean

File: StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/FollowingDto.cs (hunk 3 above)
Comment (kalachck) [reply StoreManagementService/src/PBJ.StoreManagementService.Business/Dtos/FollowingDto.cs]: don't get it, what do you mean

**Review by Genresal — APPROVED** [review]

**Review by nantonov — APPROVED** [review]

**Pull request merged**